packages/ckeditor5-dev-tests/tests/tasks/runmanualtests.js

```diff
@@ -630,6 +608,53 @@ describe( 'runManualTests', () => {
 			} );
 	} );
 
+	it( 'should set disableWatch to true if files flag is not provided', () => {
+		return runManualTests( defaultOptions )
+			.then( () => {
+				sinon.assert.calledWith( spies.scriptCompiler.firstCall, sinon.match.has( 'disableWatch', true ) );
+			} );
+	} );
+
+	it( 'should set disableWatch to false if files flag is provided', () => {
+		spies.transformFileOptionToTestGlob.returns( [] );
+
+		const options = {
+			files: [
+				'ckeditor5-classic',
+				'ckeditor-classic/manual/classic.js'
+			]
+		};
+
+		return runManualTests( { ...defaultOptions, ...options } )
+			.then( () => {
+				sinon.assert.calledWith( spies.scriptCompiler.firstCall, sinon.match.has( 'disableWatch', false ) );
+			} );
+	} );
+
+	it( 'should read disableWatch flag value even if files flag is provided', () => {
+		const options = {
+			files: [
+				'ckeditor5-classic',
+				'ckeditor-classic/manual/classic.js'
+			],
+			disableWatch: true
+		};
+
+		return runManualTests( { ...defaultOptions, ...options } )
+			.then( () => {
+				sinon.assert.calledWith( spies.scriptCompiler.firstCall, sinon.match.has( 'disableWatch', true ) );
+			} );
+	} );
+
+	it( 'should set default values for files', () => {
+		return runManualTests( defaultOptions )
+			.then( () => {
+				expect( spies.transformFileOptionToTestGlob.calledTwice ).to.equal( true );
+				expect( spies.transformFileOptionToTestGlob.firstCall.args[ 0 ] ).to.equal( '*' );
+				expect( spies.transformFileOptionToTestGlob.secondCall.args[ 0 ] ).to.equal( 'ckeditor5' );
+			} );
+	} );
```

Comment:
Please add additional test if non-default files are provided to check the arguments passed to the `transformFileOptionToTestGlob()`.